caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -0,0 +1,142 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Standard
+import os
+
+# Third Party
+from sentence_transformers import SentenceTransformer
+
+# First Party
+from caikit.core import ModuleBase, ModuleConfig, ModuleSaver, module
+from caikit.core.exceptions import error_handler
+import alog
+
+# Local
+from .embedding_tasks import EmbeddingTask
+from caikit_nlp.data_model.embedding_vectors import EmbeddingResult, Vector1D
+
+logger = alog.use_channel("TXT_EMB")
+error = error_handler.get(logger)
+
+
+@module(
+    "EEB12558-B4FA-4F34-A9FD-3F5890E9CD3F",
```

Comment:
interesting change I noticed that the id here contains all caps (plus numbers).. Other modules have small cases. There isn't a rule enforcing around it, but might be good to be consistent. 🤔

Reply:
done (in-coming)